modules/programs/neovim.nix

```
@@ -291,7 +299,8 @@ in {
 
     programs.neovim.generatedConfigViml = neovimConfig.neovimRcContent;
 
-    home.packages = [ cfg.finalPackage ];
+    home.packages = [ cfg.finalPackage ]
+      ++ (if cfg.withRemote then [ pkgs.neovim-remote ] else [ ]);
```

Small nitpick, you don't have to change this but I think there is a library function `optional` to abstract this usecase.
```suggestion
      ++ optional cfg.withRemote pkgs.neovim-remote;
```

---

That's handy!

---

Updated.